vkjokes.py

```diff
@@ -0,0 +1,35 @@
+from clients import JokeManager
+from src.schemas import VkJokesParams
+from pydantic import ValidationError
+
+import click
+
+@click.group()
+def cli():
+    """Welcome to the jokes CLI!\n
+        This script returns jokes!
+    """
+    pass
+
+@click.command()
+@click.option('--count',default=1, help='--count <int> number of jokes')
+@click.option('--lang',default='',help='--lang <lt,pl,da or ru>')
+def random(count, lang):
+    """Returns  1 random Joke!\n
+    [OPTIONS] out --count and --lang """
+    for_validation = {"count":count, "lang":lang}
+    try:
+        validated = VkJokesParams(**for_validation)
```

Comment:
Only this line risks raising a `ValidationError` in which case I would only include this line in the try-except block the return immediately after echoing the error. No need for the below lines to be wrapped too:

```
joke_manager = JokeManager(validated.count,validated.lang)
fetched_jokes =joke_manager.fetch_jokes()
```